hms-app/src/pages/Layout.tsx

```diff
@@ -41,16 +45,18 @@ const toggleColorScheme = (colorScheme: ColorScheme) =>
   colorScheme === 'dark' ? 'light' : 'dark'
 
 const Layout = () => {
+  const { instance } = useMsal()
+  let isAuthenticated = useIsAuthenticated()
+  if (!USE_AZURE_AUTH) isAuthenticated = true
```

Comment:
feel like useIsAuthenticated should be the only thing that sets isAuthenticated.  The `USE_AZURE_AUTH` seems a bit odd

Reply:
ah but thats an external thing... maybe we shuoldn't even call useIsAuthenticated if azure isn't on

Reply:
still it seems odd to have azure specific things here

Reply:
this unfortunately cannot be called conditionally as it's a hook. We might need to tweak it in the future, but for now it works with our authentication